ethereum/blockcreation/src/main/java/org/hyperledger/besu/ethereum/blockcreation/txselection/TransactionSelectionResults.java

```diff
@@ -48,27 +48,20 @@ public class TransactionSelectionResults {
       new ConcurrentHashMap<>();
 
   private long cumulativeGasUsed = 0;
-  private long cumulativeBlobGasUsed = 0;
 
   void updateSelected(
-      final Transaction transaction,
-      final TransactionReceipt receipt,
-      final long gasUsed,
-      final long blobGasUsed) {
+      final Transaction transaction, final TransactionReceipt receipt, final long gasUsed) {
     selectedTransactions.add(transaction);
     transactionsByType
         .computeIfAbsent(transaction.getType(), type -> new ArrayList<>())
         .add(transaction);
     receipts.add(receipt);
     cumulativeGasUsed += gasUsed;
-    cumulativeBlobGasUsed += blobGasUsed;
```

Comment:
why we are removing cumulativeBlobGasUsed ?

Reply:
Before this class was used to both (1)track selection results and (2)managed some state for the selectors, now the 2 tasks have been separated and in this class are only kept the selection results(1) and since the `cumulativeBlobGasUsed` is not needed as result, but only useful for the stateful selector it has been moved into the `BlobSizeTransactionSelector`